README.md

```diff
@@ -839,6 +840,63 @@ expect(element).not.toHaveTextContent('content')
 
 <hr />
 
+### `toHaveValue`
+
+```typescript
+toHaveValue(value: string | string[] | number)
+```
+
+This allows you to check whether the given form element has the specified value.
+It accepts `<input>`, `<select>` and `<textarea>` elements with the exception of
+of `<input type="checkbox">` and `<input type="radio">`, which can be meaningfully
+matched only using [`toHaveFormValue`](#tohaveformvalues).
+
+For all other form elements, the value is matched using the same algorithm
+as in [`toHaveFormValue`](#tohaveformvalues) does.
+
+#### Examples
+
+```html
+<input type="text" value="text" data-testid="input-text" />
+<input type="number" value="5" data-testid="input-number" />
+<input type="text" data-testid="input-empty" />
+<select data-testid="multiple" multiple data-testid="select-number">
+  <option value="first">First Value</option>
+  <option value="second" selected>Second Value</option>
+  <option value="third" selected>Third Value</option>
+</select>
+```
+
+##### Using document.querySelector
+
+```javascript
+const textInput = document.querySelector('[data-testid="input-text"]')
+const numberInput = document.querySelector('[data-testid="input-number"]')
+const emptyInput = document.querySelector('[data-testid="input-empty"]')
+const selectInput = document.querySelector('[data-testid="select-number"]')
+
+expect(textInput).toHaveValue('text')
+expect(numberInput).toHaveValue(5)
+expect(emptyInput).not.toHaveValue()
+expect(selectInput).not.toHaveValue(['second', 'third'])
+```
+
+##### Using dom-testing-library
+
+```javascript
+const {getByTestId} = render(/* Rendered HTML */)
+
+const textInput = getByTestId('[data-testid="input-text"]')
```

Comment:
I think these should be: `getByTestId('input-text')` and so-forth.

Reply:
Good catch! Fixed.